py3status/modules/mpris.py

```diff
@@ -510,6 +513,8 @@ def _update_metadata(self, metadata):
             # delete the file extension
             self._data["title"] = re.sub(r"\....$", "", self._data.get("title"))
 
+            self._data["nowplaying"] = metadata.get("vlc:nowplaying")
```

Comment:
I don't like hardcoded `vlc:nowplaying` string in `mpris`. Can we get a substring `nowplaying` instead? If none, return `None`... so we can support all players with that field?